int32 num
string nowNode
---
string stationNode